bin/add-exercise

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ $# -ne 1 ]; then
+    echo "Usage: bin/add-exercise <exercise-slug>"
+    exit 1
+fi
+
+command -v jq >/dev/null 2>&1 || {
+    echo >&2 "jq is required but not installed. Please install it and make sure it's in your PATH."
+    exit 1
+}
+command -v curl >/dev/null 2>&1 || {
+    echo >&2 "curl is required but not installed. Please install it and make sure it's in your PATH."
+    exit 1
+}
+
+bin/fetch-configlet
+
+# Add entry for exercise in config.json
+slug="${1}"
+name=$(echo "${slug}" | sed 's/\b\w/\u&/g')
+uuid=$(bin/configlet uuid)
+jq --arg slug "${slug}" --arg uuid "${uuid}" --arg name "${name}" \
+    '.exercises.practice += [{slug: $slug, name: $name, uuid: $uuid, practices: [], prerequisites: [], difficulty: 1}]' \
+    config.json | sed 's/"average_run_time": 10/"average_run_time": 10.0/' > config.json.tmp && mv config.json.tmp config.json
```

Comment:
add newlines to pipeline
```suggestion
    config.json \
| sed 's/"average_run_time": 10/&.0/' > config.json.tmp \
&& mv config.json.tmp config.json
```

Reply:
Wait, adding a suffix to the average run time should not be part of that pipeline, otherwise we'll eventually end up with
```none
"average_run_time": 10.0.0.0.0
```